wallet/src/util/ibc-assets.ts

```diff
@@ -0,0 +1,47 @@
+type ChainInfo = {
+  chainName: string;
+  chainId: string;
+  rpc: string;
+  addressPrefix: string;
+  explorerPath: string;
+};
+
+type AssetInfo = {
+  sourcePort: string;
+  sourceChannel: string;
+  denom: string;
+  gas?: string;
+};
+
+type IbcAssets = Record<
+  string,
+  {
+    chainInfo: ChainInfo;
+    deposit: AssetInfo;
+    withdraw: AssetInfo;
+  }
+>;
+
+export const ibcAssets: IbcAssets = {
+  IbcATOM: {
+    chainInfo: {
+      chainName: 'Cosmos Hub',
+      chainId: 'cosmoshub-4',
+      rpc: 'https://cosmoshub-rpc.stakely.io/',
```

Comment:
I wonder about fail-over. I suppose stakely's service does some of that internally, but it's still a single point of failure.

Reply:
Yes good point, it would be good to use an RPC we have some sort of relationship with, ideally an SLA.

Reply:
agree but not a blocker. please file a ticket for future work (e.g.: fall back if Stakely LB is down)

Reply:
We need autodiscovery of rpc nodes, and client qualification and selection from [the chain registry](https://github.com/cosmos/chain-registry/tree/master/cosmoshub).  Any other approach is not idea.  If we're going to hardcode something, perhaps [polkachu](https://cosmos-rpc.polkachu.com/)'s node

Reply:
not entirely sure I understand autodiscovery, but since we need it... https://github.com/Agoric/wallet-app/issues/64

Reply:
Seems like polkachu is locked down:
```
Access to fetch at 'https://cosmos-rpc.polkachu.com/' from origin 'http://localhost:3000' has been blocked by CORS policy: Request header field content-type is not allowed by Access-Control-Allow-Headers in preflight response.
```
Do you have any pointers to how autodiscovery of rpc nodes could work?

Reply:
Since this has it's own ticket I'm not considering it to be a blocker for "feat: ibc transfers"